Menü in der Toolbar

derSchumann opened the pull request:
ein und ausklappbar, aber statisch erstmal eingeklappt

derSchumann committed:
Menü in der Toolbar

ein und ausklappbar, aber statisch erstmal eingeklappt

derSchumann committed:
Icons entfernt - Menü angepasst

Jan1208 reviewed (changes requested):
- pakcage.json fehlt mit react icons
- menü zu machen beim start
- hatte für die logos oben fontawesom doer so können raus geschmissen werden

Commit:
dependencies installiert

derSchumann committed:
Fixes Menü

wenn man es wieder einklappbar haben möchte, dann muss man nur das "!" wieder einfügen.

Jan1208 reviewed (approved):
jo klasse super sache
